File: Arrays/r20.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    int i,n;
    cin >>n;
    int arr[n],brr[n];
    for(i=0;i<n;i++)
    {
        cin >> arr[i];
    }
    for(i=0;i<n;i++)
    {
        brr[i]=arr[i+1];
    }
    brr[n-1]=arr[0];
    for(i=0;i<n;i++)
    {
        cout << brr[i]<<" ";
    }
}

File: Python/Arrays/r20.py
arr = list(map(int,input().split()))
brr =[0]*len(arr)
j=0
for i in range(len(arr)):
    if(arr[i]<0):
        brr[j] +=arr[i]
        j+=1
    
for i in range(len(arr)):
    if(arr[i]>0):
        brr[j] +=arr[i]
        j+=1
        
print(brr)
        